opendcs-rest-api/src/main/java/org/opendcs/odcsapi/lrgsclient/ClientConnectionCache.java

```diff
@@ -69,9 +69,28 @@ public void removeSession(String sessionId)
 	public void setApiLddsClient(ApiLddsClient client, String sessionId)
 	{
 		CacheRecord cacheRecord = cache.computeIfAbsent(sessionId, s -> new CacheRecord());
+		ApiLddsClient lddsClient = cacheRecord.getLddsClient();
+		if(lddsClient != null)
+		{
+			lddsClient.disconnect();
+		}
 		cacheRecord.setLddsClient(client);
 	}
 
+	public void removeApiLddsClient(String sessionId)
+	{
+		CacheRecord cacheRecord = cache.get(sessionId);
```

Comment:
Looking over CacheRecord now, can it be made private? I don't see it being used outside of ClientConnectionCache.java

Reply:
Leftover from a previous iteration. Rescoped